modules/bootconf.py

```diff
@@ -14,7 +14,7 @@
 
 def configureBootloader(old, new, conffile):
     ''' Configure bootloader to use the updated rootfs '''
-    if runningDevice(conffile) == 'raspberry-pi' or runningDevice(conffile) == 'raspberry-pi2':
+    if runningDevice(conffile) == 'raspberry-pi' or runningDevice(conffile) == 'raspberry-pi2' or runningDevice(conffile) == 'raspberrypi3':
```

Comment:
Indeed. Why it happened to get this way is because I added the changes incrementally so or came more handy. :) I'll change.